doc/api/crypto.markdown

```diff
@@ -175,16 +175,27 @@ Returned by `crypto.createHmac`.
 Update the hmac content with the given `data`.  This can be called
 many times with new data as it is streamed.
 
+### hmac.validate(inputHmac)
+
+Return true if and only if the computed hmac matches the input hmac,
+provided as a [Buffer](buffer.html). This uses a timing-safe comparison.
+
+Note: `hmac` object can not be used after `validate()` method has been
+called.
+
 ### hmac.digest([encoding])
 
 Calculates the digest of all of the passed data to the hmac.  The
 `encoding` can be `'hex'`, `'binary'` or `'base64'`.  If no encoding
 is provided, then a buffer is returned.
 
+Caution: Using `digest()` directly for an HMAC will almost always
+introduce a timing attack. Prefer `validate()`, which does a
+timing-safe comparison.
+
 Note: `hmac` object can not be used after `digest()` method has been
 called.
 
-
```

Comment:
This extra line is actually intentional and should be kept, 2 spaces separate each class.